loopchain/baseservice/peer_manager.py

```diff
@@ -126,6 +126,34 @@ def __init__(self, channel_name):
         self.__leader_complain_count = 0
         self.__highest_block_height = -1    # for RS heartbeat
 
+    def peer_ids_hash(self):
+        """ It's temporary develop for Prep test net. This value will replace with Prep root hash.
+        peer_list =
```

Comment:
something strange... what is peer_list?

Reply:
It means list of rep (generator and validator)